applicationset/webhook/testdata/gitlab-event.json

```diff
@@ -16,26 +16,26 @@
       "id": 1,
       "name": "project",
       "description": "",
-      "web_url": "https://gitlab/group/name",
+      "web_url": "https://gitlab.com/group/name",
```

Comment:
Are the `.com`s here just added to make the test more realistic, or are they necessary to get the tests to pass?

Reply:
Just to make the test more realistic.